CMP1903M Assessment 1 Base Code/Analyse.cs

```diff
@@ -1,54 +1,134 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
 namespace CMP1903M_Assessment_1_Base_Code
 {
-    public static class Analyse
+    /// <summary>
+    /// Analyses text
+    /// </summary>
+    public class Analyse
     {
-        //Handles the analysis of text
+        private string _analysisText;
+        /// <summary>
+        /// Text to analyse, runs AnalyseText method when set
+        /// </summary>
+        public string AnalisisText
+        {
+            get { return _analysisText; }
+            set
+            {
+                _analysisText = value;
+                AnalyseText(_analysisText);
+            }
+        }
+        
+        /// <summary>
+        /// Keys as text elements, values as count
+        /// </summary>

```

Comment:
Could be more expressive.

e.g. contains the count of various text structures (i.e. vowels, sentences, etc)

Reply:
Thanks for pointing this out, I'll update the comments to make this clearer 👍